Source/Rhetos.Host.AspNet/RhetosAspNetCoreServiceCollectionExtensions.cs

```diff
@@ -18,23 +18,35 @@
 */
 
 using System;
+using Autofac;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Rhetos;
 using Rhetos.Host.AspNet;
+using Rhetos.Logging;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public static class RhetosAspNetCoreServiceCollectionExtensions
     {
-        public static RhetosAspNetServiceCollectionBuilder AddRhetos(this IServiceCollection serviceCollection, Action<IRhetosHostBuilder> configureRhetosHost = null)
+        public static RhetosAspNetServiceCollectionBuilder AddRhetos(
+            this IServiceCollection serviceCollection,
+            Action<IRhetosHostBuilder> configureRhetosHost = null,
+            Action<IServiceProvider, IRhetosHostBuilder> configureRhetosHostIntegration = null)
         {
             serviceCollection.AddOptions();
             if (configureRhetosHost != null)
             {
                 serviceCollection.Configure<RhetosHostBuilderOptions>(o => o.ConfigureActions.Add(configureRhetosHost));
             }
 
+            serviceCollection.Configure<RhetosHostBuilderOptions>(o => o.ConfigureIntegrationActions.Add(ConfigureLogProvider));

```

Comment:
There is no dependency for this addition. If logging is not configured in the host, Rhetos cannot be added at all?
Also breaks the principle that nothing happens automatically, so maybe user should explicitly request this kind of logging integration.